pkg/log/status.go

```diff
@@ -197,6 +197,14 @@ func Swarningf(format string, a ...interface{}) string {
 	return fmt.Sprintf(" %s%s%s", yellow(getWarningString()), suffixSpacing, fmt.Sprintf(format, a...))
 }
 
+// Warning will output in an appropriate "progress" manner
+func Experimental(a ...interface{}) {
```

Comment:
can't we just use some of the other functions for this?

Reply:
Ah no, because that'll add the whole warning symbol: ⚠️ on it. Which I believe we shouldn't need / it looks ugly.

Reply:
TBH I think the warning symbol would look appropriate

Reply:
The reason I made this function separate is so we do not have the indenting and the warning symbol, for example:

```sh
Experimental mode is enabled, use at your own risk                                                                                                                                    
                                                                                                                                                                                      
Validation                                                                                                                                                                            
 ✓  Checking Devfile compatibility [43501ns]                                                                                                                                          
 ✓  Validating Devfile [38035ns]     
```

vs

```sh
 ⚠️ Experimental mode is enabled, use at your own risk                                                                                                                                                                                                                                                                                                                          
Validation                                                                                                                                                                            
 ✓  Checking Devfile compatibility [43501ns]                                                                                                                                          
 ✓  Validating Devfile [38035ns]     
```

IMO the bottom one looks kind of odd without the newline. So I went with the first one (separate function).

Reply:
Yeah the newline makes more sense, but can we still keep the warning symbol. Looks cool and approriate TBH.